src/edgepi/adc/edgepi_adc.py

```diff
@@ -348,24 +349,24 @@ def __voltage_read(self, adc_num: ADCNum):
         return status_code, voltage_code, check_code
 
     @staticmethod
-    def __get_diff_id(mux_p: PropertyValue, mux_n: PropertyValue) -> int:
+    def __get_diff_id(mux_p: CH, mux_n: CH) -> int:
         """
         Get differential pair id number for retrieving differential pair calibration values
         """
-        #  values are the keys from adc_calib_params
-        diff_ids = {
-            DiffMode.DIFF_1.value: 8,
-            DiffMode.DIFF_2.value: 9,
-            DiffMode.DIFF_3.value: 10,
-            DiffMode.DIFF_4.value: 11,
-        }
-        diff_pair = DifferentialPair(mux_p.code, mux_n.code)
-        diff_id = diff_ids.get(diff_pair)
-        if diff_id is None:
+        # return values are the keys from adc_calib_params
+        diff_pair = DifferentialPair(mux_p, mux_n)
+        if diff_pair == DiffMode.DIFF_1.value:
+            return 8
+        elif diff_pair == DiffMode.DIFF_2.value:
+            return 9
+        elif diff_pair == DiffMode.DIFF_3.value:
+            return 10
+        elif diff_pair == DiffMode.DIFF_4.value:
+            return 11
```

Comment:
I like that you added the comment to explain the return values, but what do you think about having an enum for them? `AdcCalibParamKeys`? Maybe also mention where the keys are being used, i.e. which dictionary.

Reply:
That's a really good point, I noticed that they're not even being used in the dictionary either, so I've updated both.

https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/424/commits/0b7d64e0142a148ff470739db7491c746174b21e